tests/modules/datasets/tasks/test_dataset_catalog_indexer.py

```diff
@@ -1,53 +1,15 @@
 import pytest
 
 import dataall
```

Comment:
minor: broad import could be replaced with `dataall.modules.catalog.tasks.catalog_indexer_task.index_objects`